coalib/bears/meta.py

```diff
@@ -14,16 +15,19 @@ class bearclass(type):
     # by default a bear class has no aspects
     aspects = defaultdict(lambda: aspectlist([]))
 
-    def __new__(mcs, clsname, bases, clsattrs, *varargs, aspects=None):
+    def __new__(mcs, clsname, bases, clsattrs, *varargs, aspects=None,
+                languages=None):
         return type.__new__(mcs, clsname, bases, clsattrs, *varargs)
 
-    def __init__(cls, clsname, bases, clsattrs, *varargs, aspects=None):
+    def __init__(cls, clsname, bases, clsattrs, *varargs, aspects=None,
+                 languages=None):
         """
-        Initializes the ``.aspects`` dict on new bear classes from the mapping
-        given to the keyword-only `aspects` argument.
+        Initializes the ``.aspects`` dict and ``.languages`` array on new
+        bear classes from the mapping given to the keyword-only `aspects`
+        and `languages` arguments.
         """
         type.__init__(cls, clsname, bases, clsattrs, *varargs)
         if aspects is not None:
             cls.aspects = defaultdict(
-                lambda: aspectlist([]),
-                ((k, aspectlist(v)) for (k, v) in dict(aspects).items()))
```

Comment:
😲 What happened here? I know you fixed that in the 2nd commit https://github.com/coala/coala/pull/4111/commits/7512b0486c6af1d48ed9ebaf407c8672d251c1aa but please make every commit work on its own ;)